libnixstore/src/lib.rs

```diff
@@ -339,3 +341,20 @@ pub fn get_build_log(derivation_path: &str) -> Option<String> {
 pub fn get_nar_list(store_path: &str) -> Result<String, cxx::Exception> {
     ffi::get_nar_list(store_path)
 }
+
+fn dump_path_trampoline<F>(data: &[u8], userdata: usize) -> bool
+where
+    F: FnMut(&[u8]) -> bool,
+{
+    let closure = unsafe { &mut *((userdata as *mut std::ffi::c_void) as *mut F) };
+    closure(data)
+}
+
+#[inline]
+pub fn dump_path<F>(store_path: &str, callback: F)
```

Comment:
We need docs here, once everything is done and works

Reply:
Fixed.